main/signers/ledger/Ledger/eth.ts

```diff
@@ -57,7 +58,9 @@ export default class LedgerEthereumApp {
       domainSeparatorHex = TypedDataUtils.hashStruct('EIP712Domain', domain, types).toString('hex')
       hashStructMessageHex = TypedDataUtils.hashStruct(primaryType as string, message, types).toString('hex')
     } catch (e) {
-      throw { statusCode: 99901, message: 'Invalid typed data' }
+      const err = new Error('Invalid typed data') as unknown as DeviceError
```

Comment:
don't need to cast `e` to `unknown` here, it already has that type

Reply:
Fixed it by making `DeviceError` extend `Error`